src/js/streams/CosmosPackagesStream.js

```diff
@@ -0,0 +1,24 @@
+// to be replaced with https://github.com/dcos/dcos-ui/pull/2840
+import { Observable, Subject } from "rxjs";
+
+import CosmosPackagesStore from "#SRC/js/stores/CosmosPackagesStore";
+
+const cosmosPackageSource = Observable.timer(1000, 2000)
+  .map(() => CosmosPackagesStore.fetchRepositories())
```

Comment:
Couldn't this be `.map(CosmosPackagesStore.fetchRepositories)`?

Reply:
Unfortunately not. Because the fetchRepositories accepts a type parameter, and if I do that, I am passing the number from the timer as this parameter, but that is not correct. It is actually not gonna work because of that.

I only pass function references to map if my function is of arity 1 and is actually using the value coming from the map function `(item) => f(item)`. Otherwise, the semantics is not equivalent anymore.